test/kuttl/testmetrics/metrics-provider-dummy/00-install.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: metrics.keptn.sh/v1beta1
```

Comment:
to test the analysis bit you could add also an analysis here

Reply:
Alright.